content/docs/studio/selfhosted/configuration/tls.md

```diff
@@ -0,0 +1,83 @@
+# TLS certificates
+
+We strongly recommend deploying Studio with a TLS certificate.
+
+## Configuring up the TLS certificate and private key
+
+To configure TLS certificates to Studio, following the instructions below
+according to your installation method:
+
+<toggle>
```

Comment:
I feel like tabs/toggle is not the right thing to do here since the heart of the flow (kubectl commands) are unified. I would add an admon or collapsible section for the additional commands for EC2 instance first, instead of "splitting" the experience